src/applications/ivc-champva/10-7959C/config/submitTransformer.js

```diff
@@ -1,16 +1,72 @@
 import { transformForSubmit as formsSystemTransformForSubmit } from 'platform/forms-system/src/js/helpers';
 
+/**
+ * Convert a list of strings into an object with boolean
+ * keys indicating the presence or absence of a key. E.g.,
+ * if given list ['prop1', 'prop2'], return would be:
+ * {is_prop1: true, is_prop2: true}
+ * @param {array} listOfStr List of health insurance types (e.g., ['ppo', 'medigap'])
+ * @returns Object of boolean properties based on list of strings provided, e.g.,:
+ * {is_prop1: true, is_prop2: true}
+ */
+function listOfStrToBools(listOfStr) {
+  let retVal = {};
+  if (listOfStr !== undefined) {
+    listOfStr
+      .map(key => {
+        return { [`is_${key}`]: true };
+      })
+      .forEach(obj => {
+        retVal = { ...retVal, ...obj };
+      });
+  }
+  return retVal;
+}
```

Comment:
This is needed so that the backend can properly check the checkboxes for insurance types. It's specifically expecting keys that start with `is_`, and have boolean values.